src/library/scala/util/control/ControlThrowable.scala

```diff
@@ -12,19 +12,24 @@
 
 package scala.util.control
 
-/** A parent class for exceptions intended for flow control.
+/** A parent class for throwable objects intended for flow control.
  *
- *  Instances of `ControlThrowable` should not normally
- *  be caught. Where catch-all behaviour is required, `ControlThrowable`
- *  should be propagated; for example:
- *  {{{
- *  import scala.util.control.ControlThrowable
+ *  Instances of `ControlThrowable` should not normally be caught.
+ *
+ *  As a convenience, `NonFatal` does not match `ControlThrowable`.
  *
- *  try {
- *    // Body might throw arbitrarily
- *  } catch {
- *    case c: ControlThrowable => throw c // propagate
- *    case t: Exception        => log(t)  // log and ignore
+ *  {{{
+ *  import scala.util.control.{Breaks, ControlThrowable, NonFatal}, Breaks.{break, breakable}
+ *
+ *  breakable {
+ *    for (v <- values) {
+ *      try {
+ *        if (p(v)) break
+ *        else ???
+ *      } catch {
+ *        case NonFatal(t): Exception => log(t)  // can't catch a break
```

Comment:
😂

Reply:
Thanks for noticing. Oops, I could delete the type there.

Reply:
oh lol, I missed that. I was laughing at "can't catch a break"

Reply:
I hope so.

Reply:
Cleaned up the example. Veried by `-Xlint`!